test/winter_onboarding_2021/fleet_management/views/user_test.clj

```diff
@@ -13,3 +13,14 @@
            (hf/hiccup-text (hf/hiccup-find [:#name] (view/signup-form)))))
     (is (= "Password"
            (hf/hiccup-text (hf/hiccup-find [:#password] (view/signup-form)))))))
+
+(deftest login-form
```

Comment:
Can we also assert on the `action`? In this case, we would have to check that it isn't set?